README.md

```diff
@@ -0,0 +1,130 @@
+# Tahoe-LAFS Infrastructure
+
+The software that defines the Tahoe-LAFS project's infrastructure.
+
+## Abstract
+
+This repository was initially created in Sep 2022 as an effort to deploy and
+manage a `testgrid` server using NixOS (now retired).
+
+Since January 2025, some members of the community are seeking to re-use this repository
+to deploy and manage some new systems:
+
+- a `testgrid` server re-using the NixOS configuration written in 2022
+- a `webforge` server to [MoveOffTrac](https://github.com/tahoe-lafs/MoveOffTrac)
+
+In addition to the [Nix](https://nixos.org/) code allowing to manage the software installed on those servers,
+an [OpenToFu]() plan should handle the definition of the related infrastructure (e.g.: VMs and DNS records).
+
+## ToDo/WiP
+
+The content of this repository is mostly a work in progress (as time of writing).
+
+Here follows a list of things to do, if not already in progress (see open issues).
+
+For [Testgrid](#testgrid):
+
+- [X] Fix and rework the Nix code to deploy the current Tahoe-LAFS software
+- [ ] Implement or adapt the OpenToFu plan to cover the `testgrid` and
+  import the resource already deployed (on Hetzner and Gandi if possible)
+- [ ] Refactor the Nix code to integrate the `nixosConfiguration` with the top-level flake,
+  which will be used to manage and automated the deployment of all NixOS configurations
+
+For [Webforge](#webforge):
+
+- [ ] Implement the OpenToFu plan for Hetzner and its CI/CD workflow with GHA
+- [ ] Implement the NixOS configuration and its CI/CD workflow with GHA
+
+For the [Deployment Workflow](#deployment-workflow):
+
+- [ ] Automated the DNS configuration using OpenToFu and the relevant credentials from Gandi
+- [ ] Describe the [provisioning](#provisioning) steps with the relevant paths and snippets
+- [ ] Investigate how to allow pull request from (external) fork (with read-only token?)
+
+## How to Contribute
+
+- Read the doc and the code, consult the existing issues and ask questions (via `#tahoe-lafs` on [Libera](https://libera.chat/))
+- Describe the bug/problem or the missing info/feature in a new issue in this repository
+- Create a new branch (requires **Write** permissions in this repository) and start hacking
+- Submit pull requests (preferably one per issue) and verify the check status
+- Once a pull request as been approved and merged, verify the expected changes and provide feedback in the issue(s) if needed
+
+## Managed Systems
+
+Here is a short description of the systems managed from this repository.
+
+### Testgrid
+
+Tahoe-LAFS stand-alone grid allowing contributors to test any client software (e.g.: mobile apps).
+
+**WARN**: This server should be re-configured on regular basis,
+which means the availability of the related services and their data will NOT be guaranteed very long.
+
+Some additional information may be found in this [README](./testgrid.tahoe-lafs.org/README) file.
+
+### Webforge
```

Comment:
Is that the an alternative name (or the new name) of forgejo?  I can't find anything related when I google 'webforge'